newIDE/app/src/Utils/GDevelopServices/Asset.js

```diff
@@ -143,8 +159,16 @@ export const listAllAssets = (): Promise<AllAssets> => {
 };
 
 export const getAsset = (
-  assetShortHeader: AssetShortHeader
+  assetShortHeader: AssetShortHeader,
+  { isStagingAsset }: { isStagingAsset: boolean }
 ): Promise<Asset> => {
+  if (isStagingAsset) {
+    return axios
+      .get(
+        `${stagingBucket}/assets-database/assets/${assetShortHeader.id}.json`
```

Comment:
I'm not sure to understand why this is not just a ternary on the url?

Reply:
This is where it's a bit hacky.
Our current logic asks GDevelop backend for this URL, then makes the call.
But because the backend always returns the prod URL (no matter the environment) I went for calling the json url directly.
I'm not sure what's best:
- modifying the backend to return the "staging" bucket urls when in dev. And then we need to modify the IDE to call prod URLs by default, unless specified otherwise
- keep it like this
- change the logic to avoid making 2 calls whatever the environment, and always fetch the jsons directly

Reply:
An alternative option would be to add to the asset API a get parameter to specify the environment you want to get the bucket URL for. That way, no need to do a hack where you call the prod API to get the live buckets, since the development API could return both live and staging assets.

Reply:
I like this approach @arthuro555 👍 
I'll go for this I think

Reply:
Yep I think it's fair and allow for more flexibility for testing/development.